### PR TITLE
fix: Improve settings and fix about

### PR DESCRIPTION
**Pull request by samuveth**
Started fix for https://github.com/snapshot-labs/snapshot/issues/3571

## Changes
- Refactor `StrategiesListItem` for use in proposal, settings, about

About:
- Fix bugs with members (caused by moderator change)
- Use `StrategiesListItem` for about page strategies and add modal to view all strategies https://www.loom.com/share/43d989233e4e40d69fa4b46614d36b80

Settings:
- Start to add view only functionality (for profile, social, strategies sections)
- Use `StrategiesListItem` in strategies settings https://www.loom.com/share/f40ecb98b7fa43cfbbbb39c96bbd5f1b

### CODE REVIEW

**Comment by zzuziak** [comment]
Might be part of another PR I'm not aware of - but shouldn't we display the space info in the `About` before listing the strategies? 🤔

**Comment by samuveth** [comment]
@zzuziak which space info do you mean exactly? :)

**Comment by zzuziak** [comment]
The profile information:

image

**Review by j0hnfl0w — APPROVED** [review]
tACK

File: src/components/AvatarOverlayEdit.vue [hunk 1]
```diff
@@ -2,12 +2,18 @@
...
   loading: boolean;
   avatar?: string;
+  isViewOnly?: boolean;
 }>();
 </script>
 
 <template>
   <transition name="fade">
     <div
+      v-if="isViewOnly"
```
Comment (ChaituVR) [inline src/components/AvatarOverlayEdit.vue:12]: Can we also check if it is a demo site (demo.snapshot.org)? 
It is a recurring support question
https://github.com/snapshot-labs/snapshot/issues/3406

File: src/components/BaseButton.vue [hunk 2]
```diff
@@ -70,6 +70,7 @@ defineProps<{
 
   &:disabled {
     color: var(--border-color) !important;
+    border: 1px solid var(--border-color);
```
Comment (ChaituVR) [inline src/components/BaseButton.vue:73]: We already have this style on the button right? 🤔
Mar-17-2023 17-38-12

File: src/components/InputSwitch.vue [hunk 3]
```diff
@@ -16,7 +17,11 @@ const emit = defineEmits(['update:modelValue']);
   <div class="flex items-center space-x-2 pt-1 pr-2">
     <Switch
       :model-value="modelValue"
-      :class="modelValue ? 'bg-green' : 'bg-skin-border'"
+      :class="[
+        modelValue ? 'bg-green' : 'bg-skin-border',
+        { 'cursor-not-allowed': isDisabled }
+      ]"
+      :disabled="isDisabled"
```
Comment (ChaituVR) [inline src/components/InputSwitch.vue:24]: Oh we already have it 😅 Just getting an override

File: src/components/BaseButton.vue (hunk 2 above)
Comment (ChaituVR) [reply src/components/BaseButton.vue:73]: Oh now I know why, ignore this

File: src/components/AvatarOverlayEdit.vue (hunk 1 above)
Comment (ChaituVR) [reply src/components/AvatarOverlayEdit.vue:12]: Mar-17-2023 17-53-46
There is no cursor none for the edit button, should be fine?

File: src/components/InputSwitch.vue (hunk 3 above)
Comment (samuveth) [reply src/components/InputSwitch.vue:24]: Yep thanks fixed that

**Pull request merged**